egs2/fleurs/asr1/run.sh

```diff
@@ -46,14 +47,15 @@ if [[ "all" == *"${lang}"* ]]; then
       --valid_set "${train_dev}" \
       --test_sets "${test_set}" \
       --bpe_train_text "data/${train_set}/text" \
-      --lm_train_text "data/${train_set}/text" "$@" \
-      --local_score_opts "--score_lang_id true" "$@" 
+      --lm_train_text "data/${train_set}/text" \
+      --local_score_opts "--score_lang_id true --decode_folder decode_asr_asr_model_valid.acc.best" "$@" 
```

Comment:
I would not recommend to fix the decoder_folder here, since it can be automatically get from the scripts, can we use that instead? Otherwise, users may need multiple changes if they change any tags in the template.

Reply:
That was how I originally planned to write it, but i would need to modify `asr.sh` so that it passes the decode folder to score.sh as argument. If that is ok with you, I can do that instead.

I just wasn't sure if that was a good change since we would start passing the decode folder as an argument for all asr recipes that otherwise would not use it.

Reply:
How about following the `open_li52` recipe, which just test all the possible decoding results? As in general the scoring is very fast, it won't be taking a long time?